doc/musig-spec.mediawiki

```diff
@@ -147,7 +147,7 @@ Input:
 
 The algorithm ''Sign(secnonce, sk, aggnonce, pk<sub>1..u</sub>, m)'' is defined as:
 * Let ''R<sub>1</sub> = pointc(aggnonce[0:33]), R<sub>2</sub> = pointc(aggnonce[33:66])''; fail if that fails
-* Let ''Q = KeyAgg(pk<sub>1..u</sub>)''; fail if that fails
+* Let ''Q = point(KeyAgg(pk<sub>1..u</sub>))''; fail if that fails
```

Comment:
`point(bytes(Q)) drops the parity of Q, but do you need the parity here to know if you need to negate the signing key?

Reply:
Of course. Added `KeyAggInternal`.